src/hoc/withTheme.js

```diff
@@ -23,25 +22,46 @@ const wrapWithTheme = (Component: ReactClass<any>) => {
     static styledComponentId = 'withTheme'
 
     static contextTypes = {
-      [CHANNEL]: PropTypes.func,
+      [CHANNEL]: CONTEXT_CHANNEL_SHAPE,
```

Comment:
Oh yeah you're right, I hadn't considered that that'd use a single instance of s-c anyway :thinking: That's not as big of a deal then.